src/peergos/shared/user/UserContext.java

```diff
@@ -943,13 +943,10 @@ public boolean isJavascript() {
                 .thenCompose(wd -> wd.props.removeFromStaticData(fileWrapper, rootKey, signer, wd.hash, network, lock::complete));
     }
 
-    /**
-     * Process any responses to our follow requests.
-     *
-     * @return initial follow requests
-     */
-    public CompletableFuture<List<FollowRequest>> processFollowRequests() {
-        return network.social.getFollowRequests(signer.publicKeyHash).thenCompose(reqs -> {
+    private CompletableFuture<List<FollowRequest>> getFollowRequests() {
+        byte[] time = new CborObject.CborLong(System.currentTimeMillis()).serialize();
```

Comment:
I think i'm ok with this. browser client time should be utc and server time is utc. We haven't changed impl of System in gwt emu so it should be default impl.